.github/workflows/docs.yml

```diff
@@ -0,0 +1,37 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
nit: no need for single quotes around the branch name

Reply:
Good to know, thanks!